### PR TITLE
Update postgres-debezium-ksql-es inline with latest ksql syntax

### PR DESCRIPTION
**Pull request by big-andy-coates**
note: I've only tested the ksql statements, not the full example. It's likely the type of the keys in some of the topics has changed from a `STRING` to some other KSQL type.  I'm not sure if this will have any impact on the rest of the example.

Part of work for https://github.com/confluentinc/ksql/issues/4144

### CODE REVIEW

**Review by ybyzek — APPROVED** [review]
LGTM -- I also did not test end-to-end because it's missing a two fixes (transactions topic RF=1 and msgRate)

**Comment by big-andy-coates** [comment]
Thanks for the review @ybyzek. I don't have merge permissions on this repo, so will leave it up to you to merge this and the other PRs once you're happy.

**Pull request merged**
